sdk/iot/hub/tests/cmocka/test_az_iot_hub_client_methods.c

```diff
@@ -0,0 +1,487 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can you run the clang formatter on this file? On VSCode you can use `Shift-Alt-F`

Reply:
Done.